.travis.yml

```diff
@@ -1,7 +1,11 @@
 sudo: required
-language: Bash
+language: bash
+before_install:
+  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+  - sudo apt-get update
+  - sudo apt-get -y install docker-ce
 script:
-  - exit 0
-
+  - ./databox-test
```

Comment:
put this in a `scripts` subdirectory or similar?

Reply:
I quite like this here, might mean people get used to running it 👍